robottelo/helpers.py

```diff
@@ -262,16 +262,25 @@ def invalid_names_list():
     ]
 
 
-def invalid_values_list():
-    """List of invalid values for input testing which consists from invalid names
-    and few empty string examples. That method should be useful for CLI and API
-    tests primarily as we can pass such parameters to a server, but not in UI.
-    As we do not verify error messages, it is recommended to use current method
-    instead 'invalid_names_list' to have better coverage and prevent redundant
-    code.
+def invalid_values_list(interface=None):
+    """List of invalid values for input testing.
 
+    This returns invalid values from from invalid_names_list() and some
+    interface (api/cli/ui) specific empty string values.
+
+    :param str interface: Interface name (one of api/cli/ui).
+    :return: Returns the invalid values list
+    :rtype: list
     """
-    return ['', ' ', '\t'] + invalid_names_list()
+    # Responsibility of the developer to call right values - api/cli/ui. If
+    # not, the input will be ignored.
+    if interface not in ['api', 'cli', 'ui']:
```

Comment:
In my opinion, it will be better to use:
```
if interface == 'ui':
    return ['', ' '] + invalid_names_list()
else # api, cli, None, whatever you like
    return ['', ' ', '\t'] + invalid_names_list()

Just some code seems a little bit redundant here